Apps/CocoaPods-FCM/src/AppDelegate.swift

```diff
@@ -33,7 +33,9 @@ class AppDelegate: NSObject, UIApplicationDelegate {
         // Now that the Firebase and Customer.io SDK's are initialized, follow the rest of the required steps for the FCM push setup.
         UNUserNotificationCenter.current().delegate = self
 
-        Messaging.messaging().delegate = self
+        MessagingPushFCM.initialize { config in
+            config.autoFetchDeviceToken = true
+        }
```

Comment:
```suggestion
        // Initialize Customer.io push messaging allows 
        // the SDK to automatically send FCM push tokens to 
        // Customer.io! 
        MessagingPushFCM.initialize()
```

Adding a comment that could be helpful to customers. 

I also like the idea of using default values in the sample apps but I am open to what others prefer.

Reply:
Updated in latest commit